docs/reference/3-admin-api/installation/admin-api-1x-for-odsapi-5x-6x/admin-api-1x-docker-installation.md

```diff
@@ -0,0 +1,184 @@
+    image: edfialliance/ods-admin-api-db-admin:${ADMIN_API_DB_TAG}
+    environment:
+      POSTGRES_USER: "${POSTGRES_USER}"
+      POSTGRES_PASSWORD: "${POSTGRES_PASSWORD}"
+      API_MODE: <YOUR MODE HERE>
+    volumes:
+      - vol-db-admin:/var/lib/postgresql/data
+    restart: always
+    container_name: ed-fi-db-admin
+# ... below are other services
+```
+
+### .env Settings
+
+Add the following to your environment settings file to support Admin API. Note that when running both Admin App and Admin API, some of these settings may overlap. This is expected, and the same values can be used.
+
+**.env for Admin API**
+
+```
+ADMIN_API_TAG=<version of image to run>
+ADMIN_API_DB_TAG=<version of image to run>
+API_MODE=<API Mode Eg. SharedInstance, YearSpecific, DistrictSpecific>
+ADMIN_API_VIRTUAL_NAME=<virtual name for the Admin API endpoint>
+ODS_VIRTUAL_NAME=<virtual name for the ods endpoint>
+
+# For Authentication
+AUTHORITY=<Authentication Authority Appsetting Eg. http://localhost/${ADMIN_API_VIRTUAL_NAME}>
+ISSUER_URL=<Authentication IssuerUrl Appsetting Eg. https://localhost/${ADMIN_API_VIRTUAL_NAME}>
+SIGNING_KEY=<Authentication Signing Key (Symmetric Security Key) for Auth Tokens>
+
+# For Postgres only
+POSTGRES_USER=<default postgres database user>
+POSTGRES_PASSWORD=<password for default postgres user>
+PGBOUNCER_LISTEN_PORT=<port for pg bouncer to listen to>
+
+# The following needs to be set to specify a health check test for Admin api.
+# RECOMMENDED: To use the default internal Admin Api health check endpoint, set the variable as follows:
+ADMIN_API_HEALTHCHECK_TEST="curl -f http://${ADMIN_API_VIRTUAL_NAME}/health"
+#  To disable the health check, remove the above and instead set the variable as follows:
+# ADMIN_API_HEALTHCHECK_TEST=/bin/true
+#  To add a custom health check, consult the documentation at https://docs.docker.com/compose/compose-file/compose-file-v3/#healthcheck
+
+# The following needs to be set to specify the ODS API endpoint for Admin API to internally connect.
+# If user chooses direct connection between ODS API and Admin API within docker network, then set the api internal url as follows
+API_INTERNAL_URL = http://${ODS_VIRTUAL_NAME}
+```
+
+### Nginx / Gateway Configuration
+
+Update your nginx server configuration to include the Admin API in the reverse proxy.
+
+**default.conf.template**
+
+```
+# upstream server config...
+
+server {
+    #listen / server config...
+    #ssl_certificate config...
+
+    # other locations...
+
+    location /${ADMIN_API_VIRTUAL_NAME} {
+        client_max_body_size 20M;
+        proxy_pass http://${ADMIN_API_VIRTUAL_NAME};
+        proxy_set_header X-Forwarded-Proto $scheme;
+        proxy_set_header X-Forwarded-Host $host;
+        proxy_set_header X-Forwarded-Port 443;
+        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+    }
+}
+```
+
+## 2\. Relaunch the Docker Composition
+
+After updating the files, restart the docker composition.
+
+```
```

Comment:
Suugest using `powershell` as the language for all shell commands, unless the command is specifically for bash.